galaxy/wrapper/test-data/profiler_result2.tabular

```diff
@@ -0,0 +1,3 @@
+bin labels		-0.0Kb	0.0Kb
```

Comment:
I have a hotfix branch that fixes the problem you found previously. We can just add fixes to that, since it'll be 2.2.1.